scrapers/PuffyNetworkSite.yml

```
@@ -0,0 +1,36 @@
+name: wetandpissy
```

Should change this to PuffyNetworkSite too now.
```suggestion
name: PuffyNetworkSite
```